test/internals/reply.test.js

```diff
@@ -14,6 +14,8 @@ const {
   kReplyIsError
 } = require('../../lib/symbols')
 
+t.setTimeout(0)
```

Comment:
Everything looks good, but why this line?

Reply:
Oh, sorry, I forgot remove it after debugging